bin/verilator

```diff
@@ -2208,10 +2215,15 @@ For --make gmake, it creates:
 
     {prefix}.mk                         // Make include file for compiling
     {prefix}_classes.mk                 // Make include file with class names
+    {prefix}_hier.mk                    // Make file for hierarchy blocks
+    {prefix}_hierMkCmd.f                // Command line arguments for hierarchical
+                                           Verilation.
 
 For --make cmake, it creates:
 
     {prefix}.cmake                      // CMake include script for compiling
+    {prefix}_hierCMakeCmd.f             // Command line arguments for hierarchical
+                                           Verilation.
```

Comment:
Thanks for remembering to update this - I often forget.  

Would look a touch nicer to keep one line per file if possible without loosing critical info (section below seems to need two lines so it's ok there).

Maybe "Args" instead of "Cmd"?   Command mis-suggests to me the shell evaluates it.

```suggestion
    {prefix}_hierMkCmd.f                // Arguments for hierarchical Verilation.

For --make cmake, it creates:

    {prefix}.cmake                      // CMake include script for compiling
    {prefix}_hierCMakeCmd.f             // Arguments for hierarchical Verilation.
```

Reply:
will rename Cmd -> Args